src/Illuminate/Database/Query/Grammars/SqlServerGrammar.php

```diff
@@ -315,4 +315,22 @@ public function compileUpdate(Builder $query, $values)
 
         return trim("update {$table}{$joins} set $columns $where");
     }
+
+    /**
+     * Wrap a table in keyword identifiers.
+     *
+     * @param  string|\Illuminate\Database\Query\Expression  $table
```

Comment:
I copied it from https://github.com/laravel/framework/blob/5.2/src/Illuminate/Database/Grammar.php#L27-L33

Want me to fix it there too?